src/app/portlets/content-types/main/content-types.component.ts

```diff
@@ -104,6 +104,11 @@ export class ContentTypesPortletComponent implements OnInit {
         });
     }
 
+    public changeBaseTypeSelector(value: string) {
+        this.listing.paginatorService.setExtraParams('type', value);
+        this.listing.loadDataPaginationEvent({first: 0});
```

Comment:
I think here is better this.listing.paginatorService.getFistPage()

Reply:
`this.listing.paginatorService.getFistPage()` do not load the new results in the list, when I call ` this.listing.loadDataPaginationEvent({first: 0});` it calls `loadData` witch update the items of the listing here: https://github.com/dotCMS/core-web/blob/9f4b1db28eb6082e62ec37b19edb957dc1b27326/src/app/view/components/listing-data-table/listing-data-table.component.ts#L90